modules/vertx-swagger-codegen/src/main/resources/javaVertXServer/apiVerticle.mustache

```diff
@@ -33,11 +33,22 @@ public class {{classname}}Verticle extends AbstractVerticle {
                 {{/isString}}{{^isString}}{{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getString("{{baseName}}"), {{{dataType}}}.class);
                 {{/isString}}{{/isPrimitiveType}}{{^isPrimitiveType}}{{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getJsonObject("{{baseName}}").encode(), {{{dataType}}}.class);
                 {{/isPrimitiveType}}{{/isListContainer}}{{/allParams}}{{/hasParams}}
-                {{#returnType}}{{{returnType}}} result = service.{{operationId}}({{#hasParams}}{{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}}{{/hasParams}});
-                {{#isListContainer}}message.reply(new JsonArray(Json.encode(result)).encodePrettily());{{/isListContainer}}{{^isListContainer}}message.reply(new JsonObject(Json.encode(result)).encodePrettily());{{/isListContainer}}{{/returnType}}{{^returnType}}service.{{operationId}}({{#hasParams}}{{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}}{{/hasParams}});
-                message.reply(null);{{/returnType}}
```

Comment:
Maybe you've missed it, because mustache tags of this template are in mess :), but I think you shouldn't have remove the case where your methods don't answer anything (e.g. `addPet`).
I think you should keep the `message.reply(null)`